pytreex/block/read/conllu.py

```diff
@@ -98,5 +111,8 @@ def process_document(self, filename):
         else:
             for i in xrange(1,len(nodes)):
                 nodes[i].parent = nodes[parents[i]]
+            if len(comment):
+                root.set_attr('wild/comment', comment)
```

Comment:
Hmmm... that is probably because I haven't programmed a `set_attr` method for zones or bundles :-). But I think that it would suffice if I added the `wild` dictionary-type attribute for zones and bundles, right?

By the way, you could write `root.wild['comment'] = comment` and it would probably look better...